packages/types/src/documents/global/userGroup.ts

```diff
@@ -0,0 +1,15 @@
+import { Document } from "../document"
+import { User } from "./user"
+export interface UserGroup extends Document {
+  name: string
+  icon: string
+  color: string
+  users: User[]
+  apps: any
```

Comment:
It would be nice to have an indication here of some of the structure, is it full apps or just ids, is it an object or an array etc